requirements/checkformatting.txt

```diff
@@ -1,71 +1,52 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
 #    pip-compile --allow-unsafe requirements/checkformatting.in
 #
 black==23.3.0
-    # via -r requirements/format.txt
+    # via -r requirements/checkformatting.in
 build==0.8.0
-    # via
-    #   -r requirements/format.txt
-    #   pip-tools
+    # via pip-tools
 click==8.0.3
     # via
-    #   -r requirements/format.txt
     #   black
     #   pip-tools
 importlib-metadata==4.12.0
-    # via
-    #   -r requirements/format.txt
-    #   pip-sync-faster
+    # via pip-sync-faster
 isort==5.12.0
-    # via -r requirements/format.txt
+    # via -r requirements/checkformatting.in
 mypy-extensions==0.4.3
-    # via
-    #   -r requirements/format.txt
-    #   black
+    # via black
 packaging==23.0
     # via
-    #   -r requirements/format.txt
     #   black
     #   build
 pathspec==0.9.0
-    # via
-    #   -r requirements/format.txt
-    #   black
+    # via black
 pep517==0.12.0
-    # via
-    #   -r requirements/format.txt
-    #   build
+    # via build
 pip-sync-faster==0.0.3
-    # via
-    #   -r requirements/checkformatting.in
-    #   -r requirements/format.txt
+    # via -r requirements/checkformatting.in
 pip-tools==6.13.0
     # via
     #   -r requirements/checkformatting.in
-    #   -r requirements/format.txt
     #   pip-sync-faster
 platformdirs==2.4.0
-    # via
-    #   -r requirements/format.txt
-    #   black
+    # via black
 tomli==1.2.2
     # via
-    #   -r requirements/format.txt
     #   black
     #   build
-    #   pep517
 typing-extensions==4.0.0
-    # via
-    #   -r requirements/format.txt
-    #   black
+    # via black
 wheel==0.37.1
-    # via
-    #   -r requirements/format.txt
-    #   pip-tools
+    # via pip-tools
 zipp==3.8.1
-    # via
-    #   -r requirements/format.txt
-    #   importlib-metadata
+    # via importlib-metadata
+
+# The following packages are considered to be unsafe in a requirements file:
+pip==23.1.2
+    # via pip-tools
+setuptools==67.8.0
+    # via pip-tools
```

Comment:
I think this is due to a difference between how we compile things and how dependabot does. If you want to resolve this so we don't get so much chatter in PRs from dependabot you can remove the `--allow-unsafe` from the requirements building.